website/docs/r/notification_hub.html.markdown

```diff
@@ -74,6 +76,16 @@ A `apns_credential` block contains:
 
 ---
 
+A `browser_credential` block contains:
```

Comment:
```suggestion
A `browser_credential` supports the following:
```

Can you please update this for `apns_credential` and `gcm_credential`

Reply:
Fixed.